### ISSUE TITLE
No such file or directory

### ISSUE THREAD

**Issue by kolabearafk** [Linaqruf/kohya-trainer#83]
Hi Linaqruf, 

Thank you for the the colab notebooks. I am trying out the Kohya LoRA Dreambooth one but I am getting the " ./train.sh: line 2: sl: No such file or directory " error for some reason. Here are a few more information: 

- I didn't use Login to Huggingface hub section in 1.3.1 and the colab notebook was still able to download the Stable-Diffusion-v1-5 model in section 2.1 to /content/pre_trained_model/ as Stable-Diffusion-v1-5.ckpt .
- I didn't connect Google Drive in 1.3.2 Mount Drive section because it's written to be optional. 
- I created the train_data directory in /content/dreambooth/ using the file explorer inside the left side panel of colab. 
- I put /content/dreambooth/train_data/ as parent directory, train_repeats as 10, concept name as `<sl>` and class name as `man`.
- I clicked run for section 3.1 using the above parameters and the notebook created a new folder called `10_<sl> man` so that that folder's path is `/content/dreambooth/train_data/10_<sl> man`.
- I put all the training images (i.e. 1.png, 2.png, etc.) and the corresponding caption text files (i.e. 1.txt, 2.txt, etc.) inside the folder at path `/content/dreambooth/train_data/10_<sl> man`. 
- Then in section 5.1 to Define Important Folder, I set the pretrained_model_file_or_path as `/content/pre_trained_model/Stable-Diffusion-v1-5.ckpt`, train_folder_directory as `/content/dreambooth/train_data/10_<sl> man/` and reg_folder_directory as `/content/dreambooth/train_data/reg_data/1_man`.
- After defining specific LoRA training parameters, I start the LoRA Dreambooth training in section 5.3 but it stops and shows me the error message ./train.sh: line 2: sl: No such file or directory as mentioned above. 

Here's some more additional info from config file in case these are relevant: 

accelerate launch --config_file=/content/kohya-trainer/accelerate_config/config.yaml --num_cpu_threads_per_process=8 train_network.py --network_dim=128 --network_alpha=128 --network_module=networks.lora --learning_rate=0.0001 --text_encoder_lr=5e-05 --training_comment=this_comment_will_be_stored_in_the_metadata --lr_scheduler=constant --pretrained_model_name_or_path=/content/pre_trained_model/Stable-Diffusion-v1-5.ckpt --caption_extension=.txt --train_data_dir=/content/dreambooth/train_data/ --reg_data_dir=/content/dreambooth/train_data/reg_data/ --output_dir=/content/dreambooth/output --prior_loss_weight=1.0 --output_name=`simu_liu_<sl>` --mixed_precision=fp16 --save_precision=fp16 --save_n_epoch_ratio=3 --save_model_as=safetensors --resolution=512 --enable_bucket --min_bucket_reso=256 --max_bucket_reso=1024 --cache_latents --train_batch_size=6 --max_token_length=225 --use_8bit_adam --max_train_epochs=20 --gradient_accumulation_steps=1 --clip_skip=2 --logging_dir=/content/dreambooth/logs --log_prefix=`simu_liu_<sl>` --shuffle_caption --xformers

Please help! Thanks so much!

**Comment by Linaqruf** [comment]
helllo, try to not use <> for token,

**Comment by kolabearafk** [comment]
Thank you! Not using the <> for token really helped! 

By the way, what's the difference between instance_prompt and just prompt in the 6.2. Inference section?

**Comment by Linaqruf** [comment]
no different, final prompt is
`{instance_prompt}, {prompt} --n {negative_prompt}`

`instance_prompt` field added so you can put your trigger word without need to changing `prompt` composition, cuz people love to copy-paste prompt.

But i think it's not effective for non-danbooru model

**Comment by kolabearafk** [comment]
I see. Thanks so much for the tip and the explanation!
